src/utils/validators/index.ts

```diff
@@ -23,3 +23,15 @@ export const validatePercentage = (percent: any) => {
 
 export const validateSlack = (v: string): boolean =>
   v.match("(#|@).+") !== null;
+
+/**
+ *  validatePatchId tests if a provided id is a mongo objectId indicating that it likely belongs to a patch and not a version
+ *  @param  {string} id - the id to test
+ * @return {boolean} - true if it is a mongo objectId, false otherwise
+ */
+//
+export const validatePatchId = (id: string): boolean => {
```

Comment:
nit: can we call this isPatchId or something? validatePatchId makes it sound like it should do something or error but you're just checking if it's patch or not